litex/tools/litex_json2dts.py

```diff
@@ -542,11 +549,13 @@ def add_clkout(clkout_nr, clk_f, clk_p, clk_dn, clk_dd, clk_margin, clk_margin_e
 def main():
     parser = argparse.ArgumentParser(description="LiteX's CSR JSON to Linux DTS generator")
     parser.add_argument("csr_json", help="CSR JSON file")
+    parser.add_argument("--initrd-start", type=int, help="The location the initrd will be loaded, used to define linux,initrd-start.")
+    parser.add_argument("--initrd-size", type=int, help="The size of the initrd, used to calculate linux,initrd-end.")
     args = parser.parse_args()
 
     d = json.load(open(args.csr_json))
 
-    print(generate_dts(d))
+    print(generate_dts(d, getattr(args, "initrd_start"), getattr(args, "initrd_size")))
```

Comment:
Shouldn't `getattr(args, {"initrd_start", "initrd_size"}, None)` be used here to allow for automatic default initrd start/end handling in `generate_dts()`?

Reply:
The default value of `None` is already there in the args dict, it seems to be working that way.  Maybe I am missing something?

Reply:
Oh, you are right. I was convinced that if you don't provide `initrd_start` argument in the command line there will be no property in `args` at all and `getattr` will throw an exception. This is not the case, however, as it turns out to be `None` in such case (which plays nicely with the rest of the logic). Sorry for that.

What confused me a little was using the `getattr(x, y)` notation instead of the `x.y` one - out of curiosity: is there any practical reason for that (except for using the default value I mentioned above?).